Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -90,7 +90,7 @@ open class KolodaView: UIView, DraggableCardDelegate {
     
     public weak var delegate: KolodaViewDelegate?
     
-    public lazy var animator: KolodaViewAnimator = {
```

Comment:
This line resolved the original problem mentioned in this pull request. But then with another commit it also resolved the problem mentioned in #243.